simplecpp.cpp

```diff
@@ -190,18 +190,18 @@ void simplecpp::Token::printOut() const
 simplecpp::TokenList::TokenList(std::vector<std::string> &filenames) : frontToken(NULL), backToken(NULL), files(filenames) {}
 
 simplecpp::TokenList::TokenList(std::istream &istr, std::vector<std::string> &filenames, const std::string &filename, OutputList *outputList)
-    : frontToken(NULL), backToken(NULL), files(filenames)
+    : TokenList(filenames)
```

Comment:
delegated constructors is a C++11 feature.

Reply:
@firewave  Do you plan to update this PR?

Reply:
sure... when I am done with the anniversary cake 🎂

Reply:
@amai2012 finally...